src/main/kotlin/no/skatteetaten/aurora/boober/service/DeployBundleService.kt

```diff
@@ -103,6 +104,28 @@ class DeployBundleService(
         return createAuroraDeploymentSpec(auroraConfig, applicationId, dockerRegistry, overrideFiles, vaults)
     }
 
+    fun validateAuroraConfig(affiliation: String, function: (AuroraConfig) -> AuroraConfig = { it -> it }): AuroraConfig {
+        val deployBundle = createDeployBundle(affiliation)
+        val auroraConfig = deployBundle.auroraConfig
+
+        val newAuroraConfig = function(auroraConfig)
```

Comment:
There is no advantage to getting the AuroraConfig from calling the closure. You can just pass it directly as a parmeter to the validateAuroraConfig method.